packages/liquid-html-parser/src/stage-2-ast.ts

```diff
@@ -1915,15 +1933,15 @@ function isAcceptableDanglingMarkerClose(
     return !parentIsOfCorrectName;
   }
 
-  return isAcceptableDanglingMarker(builder, cst, currIndex, ConcreteNodeTypes.HtmlTagClose);
+  return isAcceptableDanglingMarker(builder);
 }
 
 // This function checks that the builder.current node accepts dangling nodes.
 //
 // The current logic is:
 //  - Grandparent node must be an if-like statement
 //  - Parent node must be a LiquidBranch
-//  - All sibling nodes must be of the same type (all close or all open)
+//  - All sibling nodes must be flat
```

Comment:
Not true, it's even more relaxed than this. It's the immediate parent of the dangling close must be a branch. That covers it and allows for 
```liquid
{% if cond %}
  <svg>...</svg>
  {% # works because the dangling is an immediate child of the branch %}
  </summary>
{% endif %}
```